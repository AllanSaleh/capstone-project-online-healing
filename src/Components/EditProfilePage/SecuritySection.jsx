import React from 'react';

import { useTranslation } from 'react-i18next';
import ProfileInputField from './ProfileInputField';
import LockIcon from './Images/LockIcon.svg';

export default function SecuritySection() {
  const { t } = useTranslation();
  return (
    <form className="my-16">
      <h1 className="text-3xl lg:text-title uppercase">{t('Edit.Security')}</h1>
      <ProfileInputField title={t('Edit.Pass')} type="password" id="password" icon={LockIcon} />
      <ProfileInputField
        title={t('Edit.Confirm')}
        type="password"
        id="confirm-password"
        icon={LockIcon}
      />
    </form>
  );
}
