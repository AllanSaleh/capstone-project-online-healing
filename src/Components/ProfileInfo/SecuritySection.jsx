import React from 'react';
import ProfileInputField from './ProfileInputField';
import LockIcon from './LockIcon.svg';

const SecuritySection = () => (
  <form className="my-16">
    <h1 className="text-title uppercase">security</h1>
    <ProfileInputField title="password" type="password" id="password" icon={LockIcon} />
    <ProfileInputField
      title="confirm password"
      type="password"
      id="confirm-password"
      icon={LockIcon}
    />
  </form>
);

export default SecuritySection;
