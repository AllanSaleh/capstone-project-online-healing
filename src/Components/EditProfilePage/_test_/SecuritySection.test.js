import * as React from 'react';
import * as renderer from 'react-test-renderer';
import SecuritySection from '../SecuritySection';

it('Security Section renders correctly', () => {
  const tree = renderer.create(<SecuritySection />).toJSON();
  expect(tree).toMatchSnapshot();
});
