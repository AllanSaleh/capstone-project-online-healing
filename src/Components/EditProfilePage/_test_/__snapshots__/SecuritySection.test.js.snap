// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Security Section renders correctly 1`] = `
<form
  className="my-16"
>
  <h1
    className="text-title uppercase"
  >
    security
  </h1>
  <div
    className="flex items-center my-6"
  >
    <label
      className="text-subtitle capitalize inline-block w-1/3"
      htmlFor="password"
    >
      password
    </label>
    <div
      className="w-2/3 inline-flex items-center border border-black border-opacity-20 rounded-lg pl-4"
    >
      <input
        className="w-11/12 h-full outline-none py-3"
        id="password"
        name="password"
        type="password"
      />
      <button
        type="button"
      >
        <img
          alt="profileIcon"
          src="LockIcon.svg"
        />
      </button>
    </div>
  </div>
  <div
    className="flex items-center my-6"
  >
    <label
      className="text-subtitle capitalize inline-block w-1/3"
      htmlFor="confirm-password"
    >
      confirm password
    </label>
    <div
      className="w-2/3 inline-flex items-center border border-black border-opacity-20 rounded-lg pl-4"
    >
      <input
        className="w-11/12 h-full outline-none py-3"
        id="confirm-password"
        name="confirm-password"
        type="password"
      />
      <button
        type="button"
      >
        <img
          alt="profileIcon"
          src="LockIcon.svg"
        />
      </button>
    </div>
  </div>
</form>
`;
